test/unit/testDeceasedName.js

```diff
@@ -14,25 +14,41 @@ describe('DeceasedName', () => {
         });
     });
 
-    describe('handlePost()', () => {
-        let ctx;
-        let errors;
-        let formdata;
-        let session;
-        let hostname;
-        let featureToggles;
-
-        it('should return the ctx with the deceased name and the screening_question feature toggle', (done) => {
-            ctx = {
+    describe('getContextData()', () => {
+        it('should return the ctx with the deceased address and the screening_question feature toggle on', (done) => {
+            const req = {
+                sessionID: 'dummy_sessionId',
+                session: {form: {}, featureToggles: {screening_questions: true}},
+                body: {
+                    firstName: 'Deceased FN',
+                    lastName: 'Deceased FN'
+                }
+            };
+            const ctx = DeceasedName.getContextData(req);
+            expect(ctx).to.deep.equal({
                 firstName: 'Deceased FN',
-                lastName: 'Deceased FN'
+                lastName: 'Deceased FN',
+                isToggleEnabled: true,
+                sessionID: 'dummy_sessionId'
+            });
+            done();
+        });
+
+        it('should return the ctx with the deceased address and the screening_question feature toggle off', (done) => {
```

Comment:
Please update the description for this test as it is for deceased name but it is referring to deceased address.